tensorflow/compiler/tf2tensorrt/convert/convert_nodes_test.cc

```diff
@@ -3480,6 +3480,37 @@ TEST_P(OpConverter_FP32_Test, ConvertExpandDims) {
   }
 }
 
+TEST_P(OpConverter_FP32_FP16_Test, ConvertSoftmax) {
+  // Get the NodeDef for SoftMax.
+  Scope s = Scope::NewRootScope();
+  auto input = ops::Placeholder(s.WithOpName("logits"), tf_type_);
+  auto transpose = ops::Softmax(s.WithOpName("my_softmax"), input);
+  const NodeDef& node_def = transpose.operation.node()->def();
```

Comment:
Nit: why we want to call this "transpose"?

Reply:
Corrected. It was copy-paste error.